.github/workflows/build-cli.yml

```diff
@@ -67,21 +70,27 @@ jobs:
           toolchain: stable
           target: ${{ matrix.settings.target }}
 
-      - name: Install libssl-dev for aarch64-unknown-linux-gnu
-        if: ${{ matrix.settings.target == 'aarch64-unknown-linux-gnu' }}
-        run: |
-          sudo apt-get install gcc-aarch64-linux-gnu
-
       - name: Cache cargo registry
         uses: Swatinem/rust-cache@dd05243424bd5c0e585e4b55eb2d7615cdd32f1f # v2.5.1
         with:
           key: ${{ matrix.settings.target }}-cargo-${{ matrix.settings.os }}
 
+      - name: Install Cross (aarch64-unknown-linux-gnu)
+        if: ${{ matrix.settings.target == 'aarch64-unknown-linux-gnu' }}
+        run: cargo install cross
```

Comment:
Is this cached as part of cargo caching?

Reply:
Yes